### ISSUE TITLE
Accessibility warnings or errors for insufficient color contrast

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#896]
When an author defines a `<styleDefinition>`, Doenet should verify that the color contrast is sufficiently high to pass accessibility guidelines, and emit a warning or error if not.

We should probably use the WCAG AA guideline for normal text.
